src/bundle/ui-dev/src/modules/content-tree/components/list-item/list.item.component.js

```diff
@@ -19,6 +20,15 @@ class ListItem extends Component {
         };
     }
 
+    getPrefixActions() {
+        const { prefixActions } = window.eZ.adminUiConfig.contentTreeWidget;
```

Comment:
I don't think this is the best name. Maybe `secondaryActions`, `beforeItemActions`?